examples/django/django_pwnedapi/validators.py

```diff
@@ -0,0 +1,27 @@
+from django.core.exceptions import ValidationError
+from django.utils.deconstruct import deconstructible
+from django.utils.translation import gettext_lazy as _
+
+from pwnedapi import Password
+
+
+@deconstructible
+class PwnedAPIValidator:
+    code = 'leaked'
+    message = _('Password has been seen %(count)d times before!')
+
+    def __init__(self, message=None, code=None):
+        if message is not None:
```

Comment:
I'm not that familiar with Django but is this kind of property initialization common there? I would leave these if-blocks out and just do the assignment.

Reply:
Okay, then we'll keep it. 👍